src/containers/EventListener/EventListener.stories.js

```diff
@@ -0,0 +1,192 @@
+  apiVersion: 'tekton.dev/v1alpha1',
+  kind: 'EventListener',
+  metadata: {
+    name: 'my-eventlistener',
+    namespace: 'foo',
+    creationTimestamp: '2020-01-31T16:39:21Z',
+    labels: {
+      foo: 'bar',
+      bar: 'baz'
+    }
+  },
+  spec: {
+    serviceType: 'ClusterIP',
+    serviceAccountName: 'my-serviceaccount',
+    triggers: [
+      {
+        eventListenerNamespace: 'default',
+        name: 'my-trigger',
+        bindings: [
+          { name: 'triggerbinding0' },
+          { name: 'triggerbinding1' },
+          { name: 'triggerbinding2' }
+        ],
+        template: {
+          name: 'triggertemplate'
+        },
+        interceptors: [
+          {
+            webhook: {
+              header: [
+                {
+                  name: 'header0',
+                  value: 'value0'
+                },
+                {
+                  name: 'header1',
+                  value: ['value1-0', 'value1-1', 'value1-2']
+                }
+              ],
+              objectRef: {
+                apiVersion: 'v1',
+                kind: 'Service',
+                name: 'interceptor-service0',
+                namespace: 'foo'
+              }
+            }
+          }
+        ]
+      },
+      {
+        eventListenerNamespace: 'foo',
+        name: 'my-trigger-1',
+        bindings: [{ name: 'triggerbinding1' }],
+        template: {
+          name: 'triggertemplate1'
+        },
+        interceptors: [
+          {
+            gitlab: {
+              secretRef: {
+                secretName: 'gitlab-secret',
+                secretKey: 'secret',
+                namespace: 'foo'
+              },
+              eventTypes: ['Push Hook']
+            }
+          },
+          {
+            cel: {
+              filter: "body.matches('foo', 'bar')"
+            }
+          }
+        ]
+      },
+      {
+        eventListenerNamespace: 'foo',
+        name: 'my-trigger-2',
+        bindings: [{ name: 'triggerbinding2' }],
+        template: {
+          name: 'triggertemplate2'
+        },
+        interceptors: [
+          {
+            github: {
+              secretRef: {
+                secretName: 'github-secret',
+                secretKey: 'secret'
+              },
+              eventTypes: ['push', 'pull_request']
+            }
+          }
+        ]
+      },
+      {
+        eventListenerNamespace: 'foo',
+        name: 'my-trigger-3',
+        bindings: [{ name: 'triggerbinding3' }],
+        template: {
+          name: 'triggertemplate3'
+        },
+        interceptors: []
+      }
+    ]
+  }
+};
+const match = {
+  params: {
+    eventListenerName: 'my-eventlistener'
+  }
+};
+const intl = createIntl({
+  locale: 'en',
+  defaultLocale: 'en'
+});
+const props = {
+  fetchEventListener: () => Promise.resolve(eventListener),
+  eventListener,
+  match,
+  intl
+};
+
+storiesOf('EventListenerContainer', module)
+  .add('default', () => (
+    <Router history={history}>
```

Comment:
we should use the storybook router instead of using `react-router-dom` directly to avoid a number of issues with routed components in storybook

`import StoryRouter from 'storybook-react-router';`

See https://github.com/tektoncd/dashboard/blob/master/packages/components/src/components/Header/Header.stories.js for example

Also the Container component we use for rendering stories already has an [IntlProvider configured](https://github.com/tektoncd/dashboard/blob/master/.storybook/Container.js#L23) so you shouldn't need to add your own here.